src/acl/acl.internal.ts

```diff
@@ -43,6 +53,8 @@ import {
   WithFallbackAcl,
   WithResourceAcl,
 } from "./acl";
+import { removeAll, removeIri } from "../thing/remove";
+import { predicate } from "rdf-namespaces/dist/rdf";
```

Comment:
Looks like an accidental auto-import. rdf-namespaces is only a devDependency though, so I wonder why our import tests didn't fail - possibly just because they don't import from this module?

```suggestion
```